tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplateDiagnosticTest.cs

```diff
@@ -289,6 +289,14 @@ public void TestLoopDetected()
             var lgFile = GetTemplates("LoopDetected.lg");
             var exception = Assert.Throws<InvalidOperationException>(() => lgFile.Evaluate("wPhrase"));
             Assert.Contains(TemplateErrors.LoopDetected, exception.Message);
+
+            lgFile.Options = new List<string> { "@throwOnRecursive=true" };
```

Comment:
We probably don't want customer to have to use this line to control the behavior of analyzer.  Customer will be confusing, and it locks us in where we store the options.  

Instead, if we provide that option in the paramter of the function call, we can freely change the location where it's eventually stored and used later (maybe we will create an dedicate analyzerOptions)